packages/atproto_client/models/utils.py

```diff
@@ -92,13 +97,18 @@ def _get_or_create(
         # now we are sure that this is str because it's in RECORD_TYPE_TO_MODEL_CLASS
         record_type = t.cast(str, record_type)
 
-        return get_or_create(model_data, RECORD_TYPE_TO_MODEL_CLASS[record_type], strict=strict)
+        return get_or_create(
+            model_data,
+            RECORD_TYPE_TO_MODEL_CLASS[record_type],
+            strict=strict,
+            strict_string_format=strict_string_format,
+        )
 
     if isinstance(model_data, model):
         return model_data
 
     try:
-        return model(**model_data)
+        return model.model_validate(model_data, context={'strict_string_format': strict_string_format})
```

Comment:
We could not replace constructor to the `model_validate` here without additional checks. For example, it break firehose client. Because `get_or_create` could take to `model` arg not only pydantic`s models, but also python`s data classes! So, to be able to pass `context` we should add `isinstance` with check that this is pydantic model

Reply:
i see - I updated the checks like [this](https://github.com/zzstoatzz/atproto/blob/strict-string-format/packages/atproto_client/models/utils.py#L110-L117) - what do you think?